package.json

```diff
@@ -118,12 +118,13 @@
     "source-map-loader-cli": "0.0.1",
     "strip-ansi": "4.0.0",
     "terser-webpack-plugin": "1.1.0",
-    "ts-loader": "5.3.0",
+    "ts-loader": "^5.3.3",
```

Comment:
Need to bump ts-loader. 5.3.3 makes the typescript `program` available to the transformers.